apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2634,6 +2638,55 @@ const cleanupIsolatedConversationEpic: AppEpic = (action$, state$) =>
     }),
   );
 
+const deleteChosenConversationsEpic: AppEpic = (action$, state$) =>
+  action$.pipe(
+    filter((action) =>
+      ConversationsActions.deleteChosenConversations.match(action),
+    ),
+    switchMap(() => {
+      const actions: Observable<AnyAction>[] = [];
+      const chosenConversationIds =
+        ConversationsSelectors.selectChosenConversationIds(state$.value);
+      const chosenFolderIds = ConversationsSelectors.selectChosenFolderIds(
+        state$.value,
+      );
+      const conversationIds = ConversationsSelectors.selectConversations(
+        state$.value,
+      ).map((conv) => conv.id);
+      const folders = ConversationsSelectors.selectFolders(state$.value);
+      const deletedConversationIds = uniq([
+        ...chosenConversationIds,
+        ...conversationIds.filter((id) =>
+          chosenFolderIds.some((folderId) => id.startsWith(folderId)),
```

Comment:
```suggestion
          chosenFolderIds.some((folderId) => id.startsWith(`${folderId}/`)),
```

Reply:
`chosenFolderIds` already have `/` in the end